test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/NuGet.XPlat.FuncTest.csproj

```diff
@@ -8,6 +8,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Build.Runtime" />
+    <PackageReference Include="System.Security.Cryptography.Xml" />
```

Comment:
Microsoft.CodeAnalysis.CSharp is in the wrong place for this list to have been sorted alphabetically before, but I think it's still worthwhile to move the new package to the end to make it mostly sorted.